testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationConstants.java

```diff
@@ -61,7 +61,9 @@ private static String make(String s) {
   public static final String GROUP_CLASS_LIST = make("GROUP_LIST_CLASS");
   
   public static final int DEFAULT_LOG_LEVEL = 2;
-  
+
+  public static final int DEFAULT_CONNECT_RETRIES = 360;

```

Comment:
default max retries as 360, so that the total wait time is 5 sec * 360 = 1800 sec (where 5 sec is the default socket timeout)